mgmtfn/k8splugin/driver.go

```diff
@@ -354,6 +354,15 @@ func getEPSpec(pInfo *cniapi.CNIPodAttr) (*epSpec, error) {
 	resp.Tenant = tenant
 	resp.Network = netw
 	resp.Group = epg
+
+	// Pods need to be in a group to allow policies to be applied after pod
+	// creation, if a group is not specified by the user, then place pod
+	// into a group shared across the pod's namespace, as contiv group is part
+	// of a network and network is in a k8s namespace
```

Comment:
Can you add the above four lines in the description of this PR so that reviewers know what it does.  Also, curious to know if this approach decreased the runtime of system tests? If so, by how much?